lib/java/com/google/android/material/divider/MaterialDividerItemDecoration.java

```diff
@@ -364,6 +364,16 @@ private void drawForHorizontalOrientation(@NonNull Canvas canvas, @NonNull Recyc
     canvas.restore();
   }
 
+  private int getDividerCount(RecyclerView recyclerView) {
+    int dividerCount;
+    if (recyclerView.getAdapter() != null) {
+      dividerCount = recyclerView.getAdapter().getItemCount();
+    } else {
+      dividerCount = EMPTY_ITEMS_RECYCLER_VIEW;
+    }
+    return dividerCount;
```

Comment:
Can be simplified to:

return recyclerView.getAdapter() == null ? 0 : recyclerView.getAdapter().getItemCount();